examples/viewer.py

```diff
@@ -444,15 +516,15 @@ def key_press_event(self, event: Application.KeyEvent) -> None:
             )
 
         elif key == pressed.SPACE:
-            if not self.sim.config.sim_cfg.enable_physics:
+            if not self.sim[self.active_env_idx].config.sim_cfg.enable_physics:
```

Comment:
Again, let's aim to replace all this with a helper like `self.get_sim()` or `self.default_sim` or otherwise.